algoliaUtil/algoliaUtil.go

```diff
@@ -0,0 +1,123 @@
+package algoliaUtil
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/algolia/algoliasearch-client-go/algoliasearch"
+	"github.com/comforme/comforme/common"
+)
+
+const exportAbortError string = `Export aborted: `
+
+var (
+	apiKey string
+	appId  string
+)
+
+func init() {
+	apiKey = os.Getenv("ALGOLIASEARCH_API_KEY")
+	appId = os.Getenv("ALGOLIASEARCH_APPLICATION_ID")
+}
+
+func ExportPageRecords(pages []common.Page) error {
+	if appId == "" || apiKey == "" {
+		return errors.New("Missing Algolia API keys")
+	}
+
+	client := algoliasearch.NewClient(appId, apiKey)
+
+	// Check if we need to export all page records
+	resp, err := client.ListIndexes()
+	if err != nil {
+		return errors.New(exportAbortError + err.Error())
+	}
+	indexBlob := resp.(map[string]interface{})
```

Comment:
You probably shouldn't be doing all of these type assertions. If you do:
indexBlob, ok := resp.(map[string]interface{})
if !ok {
    return fmt.Errorf(...)
}
You can not panic if a type isn't as expected.

You could also just recover from any panics and return an error.